pallets/parachain-system/src/tests.rs

```diff
@@ -457,17 +466,19 @@ fn unincluded_segment_works() {
 			|| {},
 			|| {
 				let segment = <UnincludedSegment<Test>>::get();
-				// Block 123 was popped from the segment, the len is still 2.
+				// Block 123 was popped from the segment, the len is still 1.
```

Comment:
Why? The test passes and it's 2.

Reply:
I changed the comment while debugging, but 2 is correct.

just to explain why it's correct for posterity, an inclusion delay of 1 means:
  * parent is not included (unincluded segment is 1 in pre-state)
  * block we just built isn't included (unincluded segment is 2 in post-state)